docs/_include/general-shipping/k8s.md

```diff
@@ -499,16 +499,20 @@ instrumentation.opentelemetry.io/inject-<APP_LANGUAGE>: "monitoring/logzio-monit
 By default, in multi-container pods, instrumentation is performed on the first container available in the pod spec. To fine tune which containers to instrument, add the below annotations to your pod:
 
 ```sh
-instrumentation.opentelemetry.io/inject-<APP_LANGUAGE>: "monitoring/logzio-apm-collector"
+instrumentation.opentelemetry.io/inject-<APP_LANGUAGE>: "monitoring/logzio-monitoring-instrumentation"
 instrumentation.opentelemetry.io/<APP_LANGUAGE>-container-names: "myapp,myapp2"
-instrumentation.opentelemetry.io/inject-<APP_LANGUAGE_2>: "monitoring/logzio-apm-collector"
+instrumentation.opentelemetry.io/inject-<APP_LANGUAGE_2>: "monitoring/logzio-monitoring-instrumentation"
```

Comment:
Is this only relevant here? Or do we need to edit all places where we mention the apm-collector? For example, `--set logzio-apm-collector.enabled=true`